src/quic/sessionticket.h

```diff
@@ -0,0 +1,112 @@
+#pragma once
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+#if HAVE_OPENSSL && NODE_OPENSSL_HAS_QUIC
+
+#include <crypto/crypto_common.h>
+#include <env.h>
+#include <memory_tracker.h>
+#include <uv.h>
+#include <v8.h>
+#include "data.h"
+
+namespace node {
+namespace quic {
+
+// A TLS 1.3 Session resumption ticket. Encapsulates both the TLS
```

Comment:
For folks who are not familiar with the QUIC spec, should we include links to the SPEC just like in other implementations in Node? (Applies to all)

Reply:
In one of the upcoming PRs there's going to be an extremely detailed README that provides a ton of background including links to the various specs.